honeybee_vtk/scene.py

```diff
@@ -1,281 +1,177 @@
+                Defaults to None.
+            rgba: A boolean value to set the type of buffer. A True value sets
+                an the background color to black. A False value uses the Scene object's
+                background color. Defaults to False.
             show: A boolean value to decide if the the render window should pop up.
                 Defaults to False.
 
         Returns:
-            A text string representing the path to the image.
+            A list of text strings representing the paths to the exported images.
         """
 
-        if color_range:
-            scalar_bar = vtk.vtkScalarBarActor()
-            scalar_bar.SetOrientationToHorizontal()
-            scalar_bar.SetLookupTable(color_range)
-
-            # create the scalar_bar_widget
-            legend = vtk.vtkScalarBarWidget()
-            legend.SetInteractor(self._interactor)
-            legend.SetScalarBarActor(scalar_bar)
-            legend.On()
+        return [window._export_image(
+            folder=folder, name=name + '_' + str(count), image_type=image_type,
+            image_scale=image_scale, image_width=image_width, image_height=image_height,
+            color_range=color_range, rgba=rgba, show=show)
+            for count, window in enumerate(self._windows)]
 
-        # render window
-        if not show:
-            self._window.OffScreenRenderingOn()
-        self._window.Render()
+    def export_gltf(self, folder: str, name: str = 'Camera') -> str:
+        """Export a scene to a glTF file.
 
-        image_path = pathlib.Path(folder, f'{name}.{image_type.value}')
-        writer = self._get_image_writer(image_type)
-
-        window_to_image_filter = vtk.vtkWindowToImageFilter()
-        window_to_image_filter.SetInput(self._window)
-        window_to_image_filter.SetScale(image_scale)  # image quality
+        Args:
+            folder: A valid path to where you'd like to write the gltf file.
+            name: Name of the gltf file as a text string.
 
-        # rgba is not supported for postscript image type
-        if rgba and image_type != ImageTypes.ps:
-            window_to_image_filter.SetInputBufferTypeToRGBA()
+        Returns:
+            A text string representing the path to the gltf file.
+        """
+        if self._windows:
+            return self._windows[0]._export_gltf(folder=folder, name=name)
         else:
-            window_to_image_filter.SetInputBufferTypeToRGB()
-            # Read from the front buffer.
-            window_to_image_filter.ReadFrontBufferOff()
-            window_to_image_filter.Update()
-
-        writer.SetFileName(image_path.as_posix())
-        writer.SetInputConnection(window_to_image_filter.GetOutputPort())
-        writer.Write()
-        if color_range:
-            legend.Off()
-
-        return image_path.as_posix()
+            raise ValueError(
+                'Please add at least one camera to the scene.'
```

Comment:
We don't usually use `please` in error messages. What about something like this: To be able to export an image a scene must at least have one camera assigned.

Reply:
Revised.